### DISCUSSION TITLE
Standardize/unify widget/layout configurations

### DISCUSSION THREAD

**Discussion by c0rydoras** [qtile/qtile#4293] [Ideas]
I was wondering if there are any plans to standardize or at least unify the names for widget/layout configurations. Currently, there exist both `border_width` and `borderwidth`, which can be confusing.

Here are some examples:

| Name | Other Name (Less Common) | Occurrences (of the Less Common Name) |
| --- | --- | --- |
| border_width | borderwidth | groupbox, tasklist, backends |
| color | colour | mainly found in the descriptions of widget/layout configurations ||

If wanted, i can move this next part other discussions.
Furthermore, I suggest changing the names of variables and functions to adhere to the naming conventions specified in [PEP 8](https://peps.python.org/pep-0008/#function-and-variable-names). For instance, functions like [`graphheight`](https://github.com/qtile/qtile/blob/6c92e1d947a6951b6e3d58a8330299793a59f2ad/libqtile/widget/graph.py#LL92C1-L93C75) and [`graphwidth`](https://github.com/qtile/qtile/blob/6c92e1d947a6951b6e3d58a8330299793a59f2ad/libqtile/widget/graph.py#LL88C1-L89C70) could be modified. It would also be beneficial to choose one term for each of these (e.g. color/colour & border_width/borderwidth) and then use it everywhere as it would greatly increase consistency.

**Comment by elParaguayo** [comment]
It's not a priority but I'm certainly in favour. If you can provide more examples we can look at doing this.
